### PR TITLE
Fix reload on /assets

### PR DESCRIPTION
**Pull request by marcgs**
Rename /assets route to prevent conflict with /assets directory on the web server which was causing issues on reload.

### CODE REVIEW

**Review by pkirch — APPROVED** [review]
Is the mentioned web server serving the assets directory AND the EDC Data Dashboard? 
If not, how is the conflict possible?

**Comment by marcgs** [comment]
> Is the mentioned web server serving the assets directory AND the EDC Data Dashboard? If not, how is the conflict possible?

Yes, the web server serves both the dashboard app and the `assets/` dir. The conflict arises as the app has a "/assets" route as well.

**Review by cpeeyush — APPROVED** [review]
Approved with suggestion

File: src/modules/app/app-routing.module.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ export const routes: Routes = [
     data: {title: 'Policies', icon: 'policy'}
   },
   {
-    path: 'assets',
+    path: 'my-assets',
```
[the remark this answers is not in the document]
Comment (marcgs) [reply src/modules/app/app-routing.module.ts]: I dislike "viewer" because it is not only a viewer, the page allows for creating and editing assets too

**Pull request merged**